week_03/backjoon/jclim/BOJ_2667.cpp

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
+#include <algorithm>
+#include <vector>
+#define MAX 26
+
+using namespace std;
+
+int map_size, cnt;
+char map[MAX][MAX];
+vector<int> answer;
+
+bool is_valid(int x, int y) {
+	if (x < 1 || y < 1 || x > map_size || y > map_size) {
+		return false;
+	}
+	else if (map[y][x] != '1') {
+		return false;
+	}
+	else {
+		return true;
+	}
+}
+
+void dfs(int x, int y)
+{
+	if (!is_valid(x, y)) return;
```

Comment:
```suggestion
	if !is_valid(x, y) return;
```

괄호가 불필요해 보이는데, 위와같이 해도 되지 않을까싶음

Reply:
아.. 파이썬에 너무 익숙해져있어서.. ㅈㅅ 위 코멘트는 무시 😢